airbyte-integrations/connectors/source-woocommerce/source_woocommerce/schemas/TODO.md

```diff
@@ -0,0 +1,25 @@
+# TODO: Define your stream schemas
```

Comment:
You can fairly remove this file, if you don't have dynamic json schema in your source-connector